design/parallel-commit/initial-design.md

```diff
@@ -22,9 +22,9 @@ Please see [this doc](./parallel-commit-known-issues-and-solutions.md).
 ### TiDB
 
 * The user should opt-in to parallel commit; we assume the user has opted in for all nodes for the rest of this document.
-* Each prewrite response will include a `max_read_ts`, TiDB will select the largest `max_read_ts` and add one to make the `commit_ts` for the transaction.
+* Each prewrite response will include a `min_commit_ts`, TiDB will select the largest `min_commit_ts` and add one to make the `commit_ts` for the transaction.
 * TiDB can return success to the client before sending the commit message but after receiving success responses for all prewrite messages.
-* If an operation fails because it is locked, TiDB must query the primary lock to find a list of secondaries (a `CheckTxnStatus` request). It can then send a resolve lock request to each region. If all requests succeed, it can send the commit message as above. If any fail, then it should send roll back messages to each region.
+* If an operation fails because it is locked, TiDB must query the primary lock to find a list of secondaries (a `CheckTxnStatus` request). It can then send a batch get request to each region to get all secondary locks and the `min_commit_ts` in the locks. If all locks are got, it can send the commit message as above. If any lock disappears, the primary key must have been committed or rolled back. Use `CheckTxnStatus` to get the updated information and resolve the locks.
```

Comment:
How are rollbacks initiated?

Reply:
I think it's the same as now so it's not worth mentioning. If TTL exceeds, `CheckTxnStatus` rolls back the primary lock, otherwise it returns the secondary list.